### DISCUSSION TITLE
Assistance Required with File Uploads in FilamentPHP Admin Panel

### DISCUSSION THREAD

**Discussion by wisedev99** [filamentphp/filament#14427] [Help]
### Package

Panel builder

### Package Version

^3.2

### How can we help you?

laravel 11.9 , php 8.3 , livewire 3.5

I am encountering an issue with file uploads in the FilamentPHP Admin Panel. Whenever I try to add a file, such as an image or a video, I receive a "401 Unauthorized" error, but this only occurs in the production environment.

I've tried various solutions I found on Google, but unfortunately, none of them have worked, including changes to some Nginx configurations and other adjustments.

<img width="1162" alt="Screenshot 2024-10-03 at 09 50 03" src="https://github.com/user-attachments/assets/8e0f7313-38af-458d-9e9e-2fa4160bb38b">

<img width="1508" alt="Screenshot 2024-10-03 at 09 51 04" src="https://github.com/user-attachments/assets/10eddaa3-6508-45ee-bd46-4edc9e50236a">

Could you please help me resolve this issue?

Thank you!

**Comment by veroca88** [comment]
@wisedev99 I'm getting the same error, where are you hosting your app? Were you able to solve this issue?

**Comment by mentamarindos** [comment]
anything? im facing the same issue

**Comment by ranaabdulrauf** [comment]
this might help
https://github.com/filamentphp/filament/discussions/9243
and this 
https://github.com/livewire/livewire/issues/1216

**Reply by mentamarindos** [reply]
Yeah, I’ve read those threads already and tried all the fixes people suggested, but nothing works for me 😢

**Comment by mentamarindos** [comment]
ok i finally manage to fix it
it was related to the nginx config
```
	location / {
		# First attempt to serve request as file, then
		# as directory, then fall back to displaying a 404.
		#try_files $uri $uri/ =404;
		#try_files $uri $uri/ /index.php?q=$uri&$args; #This line was causing the issue, so I commented it out. 🚫
	        try_files $uri $uri/ /index.php?$query_string; #This is the correct one ✅
	}
```

**Reply by Hemul997** [reply]
Thank you man! I had the same problem and this advice helped me a lot.
